meetings/2021-07/july-15.md

```diff
@@ -0,0 +1,734 @@
+
+JHD: awesome.
+
+CP: Yeah, I can confirm. Yeah and and they there is only one detailed Shu that maybe is important which is the fact that we have what we call undeniable intrinsics. Those are can be created from or from syntax. you modified, you modify the value of our array.prototype and you try to access the arena rate of prototype, you still the One, the intrinsic one. So it's very similar to what you just playing but even what I'm trying to say is that even for undeniable the same applies, you will get the reference that is defined in 262 to the object. That is an undeniable object intrinsic object
+
+USA:  So now that that's out of the way you can Greg’s  question, which is from stage 3. I guess I really do want to ask for that. Yes, sure. That's why we are here. Can we get a stage three on the realms proposal?
+
+JHD: That's fine with me.
+
+USA: Does anyone want to explicitly support? actually wanted to express support. Thank you.
+
+
+AKI: Looks like JWK is on the queue to say so.
+
+USA: Jack, do you want to see that load? 
+
+JWK: Yeah, I'm supporting stage 3. We have been waiting for this for too long.
+
+
+CP:Tell me about it.
+
+### Conclusion/Resolution
+- Stage 3
+- Bikeshed the name "Realm" for one more meeting.
+  - Engines (at least V8, Node, FF) agree **not** to ship unflagged until after bikeshedding
+
+
+## `getOriginals` for stage 1
+Presenter: Jordan Harband (JHD)
+
+
+USA: Yeah. And next up we have Jordan couldn't do you want to ask for stage 1? Yeah. 
+
+JHD: So even though I don't yet have a repository, we sort of sketched it out here. I would be happy make one tomorrow after I get some sleep. It's is there any reason that could not be stage one already? I would be more comfortable with sleeping more. 
+
+SYG: So, I don't want to slow down your word, I don't want them walking, so let me hear from you. Do you think that if we wait for an official explain it, I'm proposing everything through grants X3 would that slow down velocity for you?
+
+JHD: No, it's just a nice signal. I think stage one is a problem statement, right? So the specifics of the solution don't need to be explained or understood just yet but - like I'm happy to write up that explainer, it's more like, if the problem is unclear to someone such that they are not willing to or not comfortable, making it stage one without that explainer, then obviously would need to wait. But there's a possibility, we don't meet tomorrow and so I otherwise I would just say let's revisit it tomorrow. 
```

Comment:
```suggestion
JHD: No, it's just a nice signal. I think stage one is a problem statement, right? So the specifics of the solution don't need to be explained or understood just yet but - like I'm happy to write up that explainer, it's more like, if the problem is unclear to someone such that they are not willing to or not comfortable, making it stage 1 without that explainer, then obviously it would need to wait. But there's a possibility we don't meet tomorrow, otherwise I would just say let's revisit it tomorrow. 
```